source/cppfront.cpp

```diff
@@ -2853,14 +2867,15 @@ auto main(int argc, char* argv[]) -> int
     }
 
     if (cmdline.arguments().empty()) {
-        std::cout << "cppfront: error: no input files\n";
+        std::cerr << "cppfront: error: no input files\n";
```

Comment:
Outputting all errors in stderr is better regardless of whether this feature merges.

Reply:
Good catch, most of them were going to cerr but I missed these ones, thanks!